source/common/http/header_utility.cc

```diff
@@ -650,12 +650,12 @@ std::string HeaderUtility::addEncodingToAcceptEncoding(absl::string_view accept_
 }
 
 bool HeaderUtility::isStandardConnectRequest(const Http::RequestHeaderMap& headers) {
-  return headers.method() == Http::Headers::get().MethodValues.Connect &&
+  return headers.getMethodValue() == Http::Headers::get().MethodValues.Connect &&
```

Comment:
I think this is also a reason why `RequestHeaderMap` should avoid inheriting from `TraceContext`. This is a obvious mis-using of `method()`.

Reply:
Yup, nice.